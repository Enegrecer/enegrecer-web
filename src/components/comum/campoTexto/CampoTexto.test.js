import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import CampoTexto from './index';

describe('CampoTexto', () => {
  it('nÃ£o deve possuir estrutura definida ', () => {
    const onChange = jest.fn();
    const campoTexto = mount(<CampoTexto maxLen={3} onChange={onChange} />);
    const campoTextoJson = toJson(campoTexto);
    expect(campoTextoJson).toMatchSnapshot();
  });
});

