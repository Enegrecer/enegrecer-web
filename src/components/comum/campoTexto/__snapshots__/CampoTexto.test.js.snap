// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampoTexto n√£o deve possuir estrutura definida  1`] = `
<campoTexto
  divClasse=""
  id=""
  inputClasse=""
  label=""
  maxLen={3}
  onChange={[Function]}
  placeholder=""
  type=""
>
  <div
    className=""
  >
    <input
      className=""
      id=""
      maxLength={3}
      onChange={[Function]}
      type=""
    />
    <label
      className="active"
      htmlFor=""
    />
  </div>
</campoTexto>
`;
